libs/components/avatar/src/lib/modules/avatar/avatar.component.ts

```diff
@@ -55,7 +55,7 @@ export class SkyAvatarComponent {
   }
 
   /**
-   * Specifies an image to display to identify a record. This property is
+   * The image to display to identify a record. This property is
```

Comment:
```suggestion
   * The image to identify a record. This property is
```
Minor edit to make this read more naturally. "to display" reads awkwardly and seems unnecessary.